source/includes/options-mongo.yaml

```diff
@@ -350,4 +350,57 @@ description: |
    .. versionadded:: 3.2
 
    Disables use of the JavaScript engine's JIT compiler.
+---
+program: mongo
+name: disableJavaScriptProtection
+directive: option
+optional: true
+args: null
+description: |
+   .. versionadded:: 3.4
+
+   Allows documents containing fields of type
+   :ref:`javascript<bson-types>` and
+   :ref:`javascriptWithScope<bson-types>` to be automatically
+   marshalled to JavaScript functions in the :program:`mongo`
```

Comment:
Are we marshalling the whole documents to JavaScript functions?  Because that's what this sentence states.